libphobos/libdruntime/core/sys/posix/stdio.d

```diff
@@ -279,7 +279,7 @@ unittest
 { /* Note: open_memstream is only useful for writing */
     import core.stdc.string : memcmp;
     char* ptr = null;
-    char[] testdata = ['h', 'e', 'l', 'l', 'o', 0];
+    char[6] testdata = ['h', 'e', 'l', 'l', 'o', 0];
```

Comment:
It's not really libtool requiring this, but I'd like to avoid making exceptions for these files. We also do have to compile the files to run the unit tests (we didn't test these modules until now)

The changes are backports.